http/transport.go

```diff
@@ -32,13 +32,11 @@ func NewAPIRouter() *mux.Router {
 	r.NewRoute().Name("ListServices").Methods("GET").Path("/v6/services").Queries("namespace", "{namespace}") // optional namespace!
 	r.NewRoute().Name("ListImages").Methods("GET").Path("/v6/images").Queries("service", "{service}")
 
-	r.NewRoute().Name("UpdateImages").Methods("POST").Path("/v6/update-images").Queries("service", "{service}", "image", "{image}", "kind", "{kind}")
```

Comment:
We'll have to take care to reinstate these routes in the service code, since our contract there is different*.

*Here we expect people to download a `fluxctl` compatible with their daemon, though perhaps we shouldn't. In the service, we have to support `fluxctls` in a range of versions -- so while we can modify the api interfaces post-hoc, we oughtn't remove routes that will still be called.

Reply:
Fortunately if this route doesn't exist after revendoring in the service code it'll panic when building the handler :)

Reply:
Panic or fail to compile?

Reply:
Yes it'll probably panic. We'd have to change this around a fair bit to induce a compilation error (types, what are those).

Perhaps ... perhaps the router definitions should be versioned as well? We've opened a can of worms here ..

Reply:
We should probably version everything about the API sooner rather than later.

I'm not sure we can introduce a compile error at all, how would you go about it?

Reply:
I figured out why this bothers me -- people shouldn't need to download a new `fluxctl` after we've refactored. I suggest we reinstate the `UpdateImages` and `UpdatePolicy` endpoints, and implement them in terms of `UpdateManifests`.

NB they don't need to appear in the `Server` interface, since old `fluxctl` will have been built with the old interface, and new `fluxctl` will just use `UpdateManifests`.